### ISSUE TITLE
Binary Sensor does not support platform setup

### ISSUE THREAD

**Issue by coreGreenberet** [JoDehli/PyLoxone_beta#5]
HomeAssistant 2023.1.0

```
Logger: homeassistant.components.binary_sensor
Source: helpers/entity_platform.py:198
Integration: Binärsensor (documentation, issues)
First occurred: 14:10:27 (1 occurrences)
Last logged: 14:10:27

The loxone platform for the binary_sensor integration does not support platform setup. Please remove it from your config.
```

**Comment by JoDehli** [comment]
I work at the moment (all afternoon) on a new version of pyloxone. Had no time to fix also the pyloxone beta. Sorry. But I think when pyloxone is finished it is much better than before and it will work also for the gen2.

**Comment by JoDehli** [comment]
Maybe I can fix the mentioned problems this morning. Should be no big deal. But for the future I want to archive the beta and you should use the main branch. Then you get all updates automatically.

**Comment by JoDehli** [comment]
@coreGreenberet fixed in 0.0.9
